httprequest_blueprints/download_file.py

```diff
@@ -122,6 +122,12 @@ def main():
             authorization_header)
 
     download_file(url, destination_name, headers, params)
+    written_file_size = os.path.getsize(destination_name)
+    print(
+        f'The downloaded file contained {written_file_size/1000000}MB of data.')
```

Comment:
Will this display correctly? Maybe we want something more like `f'{diff:.5f}'` to avoid really, really small numbers.

Reply:
From my testing, yes. Resulting output looks like this. 
<img width="359" alt="image" src="https://user-images.githubusercontent.com/3869123/153509603-2dc8c9e5-68c7-4615-a0b8-59ff3a9ece0e.png">

Reply:
Honestly, that looks fine. _(with Axolo)_

Reply:
Just wanted to check. _(with Axolo)_

Reply:
Why not print the total number of bytes? 

Also, below, you are going to get the above to print "... contained 0MB of data" and the below output.